### DISCUSSION TITLE
Release v0.5.2

### DISCUSSION THREAD

**Discussion by Iamlooker** [Droid-ify/client#221] [Announcements]
# Join Our New[ Discord](https://discord.gg/4zrrDdB3) Server 🎉 

## UI Changes 📸 
- New App Icon
- Add: Material You support
- Add: Edge-to-Edge support (Android 11+)
- Add: Snackbar if the address is not valid
- Add: Snackbar to represent Internet Connectivity
- Add: Indicator for installed apps
- Add: Favourites list [In menu]
- Add: App bar controls
- Add: New Error icon
- Improve: `Repository Details` page
- Improve: `Repository List` page
- Fix: Theme Changing
- Fix: App bar not scrolling
- Fix: Navigation bar color

## Core 📖 
- Add: Favourites list in menu
- Add: `Never` cleanup option
- Add: Session and FluffyChat Repository
- Remove: Some dead repositories
- Remove: Support for legacy `index.xml` (For security purposes)
- Fix: Many installer bug
- Fix: App not visible in Android TV
- Fix: A very annoying random crash
- Fix: Shizuku and Root Installer

## Misc ❔ 
- Better Install notification icon
- Click App Icon to switch Author and Package name
- Use legacy installer on first time MiUi users
- Shizuku dropped support for pre-Android 11
- Fix: Install dialog not visible when notification is clicked
- Fix: Crash on bad repository address
- Fix: No back button in settings page

## What's Changed
* Fix wrong signature certificates by @Iamlooker in https://github.com/Iamlooker/Droid-ify/pull/182
* Add debug apk workflow by @daBroma in https://github.com/Iamlooker/Droid-ify/pull/185
* create workflow for updating locales by @C10udburst in https://github.com/Iamlooker/Droid-ify/pull/172
* Allow a setting to control top app bar by @Iamlooker in https://github.com/Iamlooker/Droid-ify/pull/188
* Add simple edge-to-edge for Android 11+ by @Iamlooker in https://github.com/Iamlooker/Droid-ify/pull/190
* Removed warnings after building debug apk by @Daviteusz in https://github.com/Iamlooker/Droid-ify/pull/191
* Fix typo by @itec78 in https://github.com/Iamlooker/Droid-ify/pull/202
* Add FluffyChat and Threema Libre repository by @linux2good in https://github.com/Iamlooker/Droid-ify/pull/210
* Fix README.md by @linux2good in https://github.com/Iamlooker/Droid-ify/pull/213
* Implementation of dynamic "Material You" colors by @t3pfaffe in https://github.com/Iamlooker/Droid-ify/pull/204
* Add Session messanger repository, add/fix descriptions by @linux2good in https://github.com/Iamlooker/Droid-ify/pull/214
* Add tv banner (Closes #169, Touches #60) by @Iamlooker in https://github.com/Iamlooker/Droid-ify/pull/217
* Delete Ungoogled Chromium by @linux2good in https://github.com/Iamlooker/Droid-ify/pull/220

## New Contributors
* @daBroma made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/185
* @C10udburst made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/172
* @Daviteusz made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/191
* @itec78 made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/202
* @linux2good made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/210
* @t3pfaffe made their first contribution in https://github.com/Iamlooker/Droid-ify/pull/204

**Full Changelog**: https://github.com/Iamlooker/Droid-ify/compare/v0.5.1...v0.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Iamlooker/Droid-ify/releases/tag/v0.5.2'>Release v0.5.2</a>.</em>

**Comment by Sdash24** [comment]
Can't install (updating from v0.5.1)

**Reply by t3pfaffe** [reply]
If you are using the F-Droid version and trying to install with the APK from this release or from the IzzyOnDroid repository it will fail to update. This is because this APK is not signed with the same key as it was when it was compiled by F-Droid in the previous version. To fix this you can either wait for it to be complied and released on the F-Droid repo or uninstall & reinstall it with the new APK.
